### ISSUE TITLE
Xcode 12 warns about iOS Simulator Deployment Target being too low

### ISSUE THREAD

**Issue by danielsaidi** [Quick/Quick#998]
- [ x] I have read [CONTRIBUTING](https://github.com/Quick/Quick/blob/master/CONTRIBUTING.md) and have done my best to follow them.

### What did you do?

Build a project with dependencies to Quick and Nimble.

### What did you expect to happen?

I sure expect problems every year, but Q&N worked great...

### What actually happened instead?

...but there is a warning about the frameworks targeting iOS 8.0.

```
The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99.
```
### Environment

List the software versions you're using:

 - Quick: 3.0.0
 - Nimble: 8.1.0
 - Xcode Version: 12 beta 2
 - Swift Version: 5.3

Please also mention which package manager you used and its version. Delete the
other package managers in this list:

 - Swift Package Manager *5.3.0*

**Comment by ikesyo** [comment]
Hmm we'll need another major bump 😫

**Comment by danielsaidi** [comment]
😢

But really? Although dropping iOS 8 would semantically require a major bump, it's actually Xcode that stops supporting iOS 8, not Quick. So...is a major bump really required?

**Comment by dwhitla** [comment]
The issue here is actually that your package config contradicts itself. The Quick (and Nimble) `Package.swift` state the `.iOS(.v8)` is supported while your direct dependency CwlPreconditionTesting has a minimum supported iOS version of 9. This unsatisfiable version constraint is detected and results in a hard fail on Xcode 12 (Swift Tools 5.3.0).
You MUST drop support for iOS 8 or change the version constraint on the CwlPreconditionTesting dependency (currently `..<2.0.0`).

**Comment by ikesyo** [comment]
Closed by #1013.
